app/Locale/Spanish.php

```diff
@@ -426,34 +426,34 @@ public function get_locale(): array
 			'UPLOAD_FAILED_ERROR' => 'Subida fallida. ¡El servidor devolvió un error!',
 			'UPLOAD_FAILED_WARNING' => 'Subida fallida. ¡El servidor devolvió una advertencia!',
-			'UPLOAD_CANCELLED' => 'Cancelled',
+			'UPLOAD_CANCELLED' => 'Cancelado',
 			'UPLOAD_SKIPPED' => 'Saltado',
-			'UPLOAD_UPDATED' => 'Updated',
-			'UPLOAD_IMPORT_SKIPPED_DUPLICATE' => 'This photo has been skipped because it\'s already in your library.',
-			'UPLOAD_IMPORT_RESYNCED_DUPLICATE' => 'This photo has been skipped because it\'s already in your library, but its metadata has been updated.',
+			'UPLOAD_UPDATED' => 'Actualizado',
+			'UPLOAD_IMPORT_SKIPPED_DUPLICATE' => 'Esta foto ha sido omitida porque ya está en tu biblioteca.',
+			'UPLOAD_IMPORT_RESYNCED_DUPLICATE' => 'Esta foto se omitió porque ya está en tu biblioteca, pero sus metadatos se actualizaron.',
 			'UPLOAD_ERROR_CONSOLE' => 'Por favor, eche un vistazo a la consola de su navegador para más detalles.',
 			'UPLOAD_UNKNOWN' => 'El servidor devolvió una respuesta desconocida. Por favor, eche un vistazo a la consola de su navegador para más detalles.',
 			'UPLOAD_ERROR_UNKNOWN' => 'Subida fallida. ¡El servidor devolvió un error desconocido!',
-			'UPLOAD_ERROR_POSTSIZE' => 'Upload failed. The PHP post_max_size may be too small! Otherwise check the FAQ.',
-			'UPLOAD_ERROR_FILESIZE' => 'Upload failed. The PHP upload_max_filesize may be too small! Otherwise check the FAQ.',
+			'UPLOAD_ERROR_POSTSIZE' => 'Subida fallida. ¡El PHP post_max_size puede ser demasiado pequeño! De lo contrario, consulta las preguntas frecuentes.',
+			'UPLOAD_ERROR_FILESIZE' => 'Subida fallida. ¡El PHP upload_max_filesize puede ser demasiado pequeño! De lo contrario, consulta las preguntas frecuentes.',
 			'UPLOAD_IN_PROGRESS' => '¡Lychee está subiendo actualmente!',
 			'UPLOAD_IMPORT_WARN_ERR' => 'La importación ha finalizado, pero devolvió advertencias o errores. Por favor, eche un vistazo al registro (Configuración -> Mostrar registro) para obtener más detalles.',
 			'UPLOAD_IMPORT_COMPLETE' => 'Importación completa',
 			'UPLOAD_IMPORT_INSTR' => 'Ingrese el enlace directo a una foto para importarla:',
 			'UPLOAD_IMPORT' => 'Importar',
 			'UPLOAD_IMPORT_SERVER' => 'Importando desde el servidor',
 			'UPLOAD_IMPORT_SERVER_FOLD' => 'Carpeta vacía o no hay archivos legibles para procesar. Por favor, eche un vistazo al registro (Configuración -> Mostrar registro) para obtener más detalles.',
-			'UPLOAD_IMPORT_SERVER_INSTR' => 'Import all photos, folders and sub-folders located in the folders with the following absolute paths (on server). Paths are space separated, use \\ to escape a space in a path.',
-			'UPLOAD_ABSOLUTE_PATH' => 'Absolute path to directories, space separated',
+			'UPLOAD_IMPORT_SERVER_INSTR' => 'Importe todas las fotos, carpetas y subcarpetas ubicadas en las carpetas con las siguientes rutas absolutas (en el servidor). Las rutas están separadas por espacios, use \\ para escapar de un espacio en una ruta.',
```

Comment:
Is "use" still correct here?

Reply:
yes, it is. it is the formal Spanish tense of the verb. it is the same in both languages although the "sense" is a little different